test/focusable-arrowkeys-behavior.html

```diff
@@ -66,6 +66,9 @@
 							simpleFixture.arrowKeyFocusablesOnBeforeFocus = function(elem) {
 								return new Promise(function(resolve) {
 									expect(elem).to.equal(focusables[keyInteractions[i].endIndex]);
+									if (focusables[keyInteractions[i].startIndex] === focusables[keyInteractions[i].endIndex]) {
```

Comment:
I added this check here due to `testKeyInteractions` expecting a `focus` event to be fired (which didn't seem to be the case, despite returning the same element [here](https://github.com/Brightspace/d2l-polymer-behaviors-ui/pull/30/files#diff-39de0138258dd6c3cae86c606674984aR182), as opposed to null

I don't think this is the best idea, and I'll write individual tests instead.